src/modules/service-catalog/components/service-catalog-list-item/ServiceCatalogListItem.tsx

```diff
@@ -0,0 +1,88 @@
+import type { ServiceCatalogItem } from "../../data-types/ServiceCatalogItem";
+import ShapesIcon from "@zendeskgarden/svg-icons/src/16/shapes-fill.svg";
+import styled from "styled-components";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+
+const StyledItem = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: relative;
+  gap: ${(props) => props.theme.space.sm};
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  border-radius: ${(props) => props.theme.borderRadii.md};
+  cursor: pointer;
```

Comment:
Instead of using `cursor: pointer` I would already transform this into an `a` tag even if we don't have the other page right now, just to have the proper styles already implemented. We can have an `a` with `href="#"`

Reply:
Ok changed to `a` tag